twine/commands/upload.py

```diff
@@ -30,6 +34,26 @@
 def skip_upload(
     response: requests.Response, skip_existing: bool, package: package_file.PackageFile
 ) -> bool:
+    """Skip uploading a package.
+
+    Return Boolean type according to the status code that responded by the repository or
+    the argument passed by the user when trying to upload the package(s).
+
+    If ``skip_existing`` is set to ``True``, then return ``False``.
+    If status code 400, 403, 409 is responded by the repository, return ``True``.
+
+    :param requests.Response response:
```

Comment:
This will require setting up `intersphinx` in order to work correctly. Can you update the sphinx conf file?

Reply:
Could I use something like this?
```rst
:param :py:class:requests.Response response:
   Get the response from the repository.
```

Reply:
You can't do it there but if you do it separately that works.